fastlane/Fastfile

```diff
@@ -72,7 +72,7 @@ ENV["PUBLIC_CONFIG_FILE"]=File.join(PROJECT_ROOT_FOLDER, "config", "Version.Publ
 ENV["INTERNAL_CONFIG_FILE"]=File.join(PROJECT_ROOT_FOLDER, "config", "Version.internal.xcconfig")
 ENV["DOWNLOAD_METADATA"]="./fastlane/download_metadata.swift"
 ENV["PROJECT_ROOT_FOLDER"]=PROJECT_ROOT_FOLDER + "/"
-
+ENV["APP_STORE_STRINGS_FILE_NAME"]="AppStoreStrings.po"
```

Comment:
Out of scope.

While looking the code and the history (see [previous comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/16038/files#r589863179)) I noticed in WooCommerce iOS [this file has the `.pot` extension](https://github.com/woocommerce/woocommerce-ios/blob/develop/WooCommerce/Resources/AppStoreStrings.pot).

From what I understand from [Wikipedia's explanation of how `gettext` works for localizations](https://en.wikipedia.org/wiki/Gettext), `.pot` stands for Portable Object Template and "contains a list of all the translatable strings extracted from the sources" while `.po` stands for Portable Object and it's a file that the translator derives "from the template using the `msginit` program, then fills out the translations". `msginit` "initializes the translations".

I'm not 100% sure of what that all means in practical terms for us, but I suspect `.pot` would be the correct extension.